### DISCUSSION TITLE
Asset field upload folder structure based on the files you upload

### DISCUSSION THREAD

**Discussion by kevinmu17** [craftcms/cms#12063] [Q&A] [answered]
Is there a way to get the uploaded file data when uploading directly to an asset field (so without the popup)?

I'm trying the following but without success:
Default folder location:
`{ UPLOADEDFILE.getMimeType() == 'video/mp4' ? 'videos' }`

**Comment by brandonkelly** [comment, marked as answer]
Not currently, sorry.
